### PR TITLE
Add recipe for pytest-picked

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
This is a `noarch: python` recipe, so I think this is now ready for review/merge. 👍

File: recipes/pytest-picked/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
```
Comment (marcelotrevisani) [inline recipes/pytest-picked/meta.yaml]: Hi @nicoddemus , could you please change this line to:
```
"{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

This is the new default for conda to install python projects

File: recipes/pytest-picked/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools
+    - pytest >=3.5.0
+  run:
+    - python >=3.5
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/pytest-picked/meta.yaml]: Does this project use ``setuptools`` as a run requirement? or just for build?

**Comment by marcelotrevisani** [comment]
Hi @nicoddemus , I did some comments in the recipe, just take a look, after that I think this is good to go.

Thanks to put this in ``conda-forge``

File: recipes/pytest-picked/meta.yaml (hunk 2 above)
Comment (nicoddemus) [reply recipes/pytest-picked/meta.yaml]: Ah good catch, totally missed that. Thanks!

File: recipes/pytest-picked/meta.yaml (hunk 1 above)
Comment (nicoddemus) [reply recipes/pytest-picked/meta.yaml]: Didn't know that changed, thanks!

**Comment by marcelotrevisani** [comment]
Thanks! I'm going to merge this! :)

**Pull request merged**